src/Controller/QuestionController.php

```diff
@@ -87,4 +94,82 @@ public function result(): string
             ['message' => $message]
         );
     }
+
+    /**
+     * Creates the page that allows users to enter their question
+     * and the number of choices they want to add
+     * @return string
+     */
+    public function add(): string
+    {
+        $error = "";
+
+        // Check if userQuestion and numberOfChoices exist in $_POST
+        if (isset($_POST['userQuestion']) && isset($_POST['numberOfChoices'])) {
+            $userQuestion = trim($_POST["userQuestion"]);
+
+            // Check if question isn't too long
+            if (strlen($userQuestion) > self::QUESTION_MAX_LENGTH) {
+                $error = "Ta question est trop longue! Elle doit faire maximum "
+                . self::QUESTION_MAX_LENGTH . " caractères.";
+                return $this->twig->render(
+                    'Question/add.html.twig',
+                    ['error' => $error]
+                );
+            }
+
+            // Check if question isn't too short
+            if (strlen($userQuestion) <= self::QUESTION_MIN_LENGTH) {
+                $error = "On sait que la taille fait pas tout mais ta question est trop courte!";
+                return $this->twig->render(
+                    'Question/add.html.twig',
+                    ['error' => $error]
+                );
+            }
+
+            $numberOfChoices = filter_input(
+                INPUT_POST,
+                'numberOfChoices',
+                FILTER_VALIDATE_INT,
+                ["options" => [
+                    "min_range" => self::MIN_NUMBER_OF_CHOICES,
+                    "max_range" => self::MAX_NUMBER_OF_CHOICES]
+                ]
+            );
+
+            // Check if number of questions is a valid number
+            if ($numberOfChoices === false || $numberOfChoices === null) {
+                $error = "Tu dois avoir entre "
+                . self::MIN_NUMBER_OF_CHOICES . " et "
+                . self::MAX_NUMBER_OF_CHOICES . " choix!";
+
+                return $this->twig->render(
+                    'Question/add.html.twig',
+                    ['error' => $error]
+                );
+            }
+
+            // Put question and number of choices into $_SESSION
+            if (isset($_SESSION['userQuestion'])) {
+                unset($_SESSION['userQuestion']);
+            }
```

Comment:
Seems useless because you will replace the $_SESSION['userQuestion'] value just afterwards